### ISSUE TITLE
Tracker crashes

### ISSUE THREAD

**Issue by pderrenger** [glenn-jocher/ultralytics#7]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### Bug

Tracking crashes for me. How can I customize the tracker YAML with YOLOv8m?

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
@pderrenger hello, 

Can you please provide me with more information about your issue? Also, can you let me know if you have tried searching for a solution already? If not, you can try checking out our documentation at https://docs.ultralytics.com for more information on how to customize the tracker YAML with YOLOv8m.

Thanks!

**Comment by pderrenger** [comment]
Are there tracker examples in the docs?

**Comment by glenn-jocher** [comment]
@pderrenger yes, there are tracker examples in the documentation for YOLOv8. You can find them in the YOLOv8 Tracker section of our documentation at https://docs.ultralytics.com/reference/tracker/track/. Additionally, you can also find more information on how to customize the tracker YAML with YOLOv8m in our documentation. Hope that helps!

**Comment by pderrenger** [comment]
Are there Pose examples you could show me?

**Comment by glenn-jocher** [comment]
@pderrenger certainly! You can find Pose examples for YOLOv5 in our documentation at https://docs.ultralytics.com/yolov5/tutorials/custom_pose/. This tutorial will guide you through the process of training a custom Pose model based on the YOLOv5 architecture, and will also provide examples for inference and validation. Additionally, the YOLOv5 Pose section of our documentation at https://docs.ultralytics.com/yolov5/yolov5_pose/ provides more information and references for Pose tasks using YOLOv5. Hope that helps!
